mongodbatlas/data_source_mongodbatlas_projects.go

```diff
@@ -160,17 +204,42 @@ func flattenProjects(ctx context.Context, conn *matlas.Client, projects []*matla
 		for k, project := range projects {
 			teams, _, err := conn.Projects.GetProjectTeamsAssigned(ctx, project.ID)
 			if err != nil {
-				fmt.Printf("[WARN] error getting project's teams assigned (%s): %s", project.ID, err)
+				diagWarning := diag.Diagnostic{
+					Severity: diag.Warning,
+					Summary:  "Error getting project's teams assigned",
+					Detail:   fmt.Sprintf("Error getting project's teams assigned (%s): %s", project.ID, err),
+				}
+				diags = append(diags, diagWarning)
 			}
 
 			apiKeys, err := getProjectAPIKeys(ctx, conn, project.OrgID, project.ID)
 			if err != nil {
-				fmt.Printf("[WARN] error getting project's api keys (%s): %s", project.ID, err)
+				diagWarning := diag.Diagnostic{
+					Severity: diag.Warning,
+					Summary:  "Error getting project's api keys",
+					Detail:   fmt.Sprintf("Error getting project's api keys (%s): %s", project.ID, err),
+				}
+				diags = append(diags, diagWarning)
```

Comment:
This change seems the root cause of the issue https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1371